example/models_directory/configuration_interaction.json

```diff
@@ -0,0 +1,15 @@
+{
+    "tier1": "physics-based",
+    "tier2": "quantum-mechanical",
+    "tier3": "ab initio",
+    "type": "configuration interaction",
+    "subtype": "cisd",
+    "modification": "multi-reference",
+    "augmentation": null,
```

Comment:
1. We might as well omit the key if there's nothing to be stored there, shall we?
2. I thought modifications would be an array with multiple members, should we make it singular instead?